civicase.php

```diff
@@ -577,7 +577,7 @@ function civicase_civicrm_navigationMenu(&$menu) {
    *   Array(string $oldUrl => string $newUrl).
    */
   $rewriteMap = [
-    'civicrm/case?reset=1' => 'civicrm/case/a/#/case',
+    'civicrm/case?reset=1' => 'civicrm/case/a/#/case?case_type_category=cases',
```

Comment:
Since we are already doing some renaming here. Why don't we just do the renaming for manage cases here and we won't need to have Upgraders or Setup.

i.e
```php
'civicrm/case/a/#/case/list' => ''civicrm/case/a/#/case/list?cf=%7B"case_type_category":"cases"%7D'
```

Reply:
As discussed, this is not implemented.